### PR TITLE
[ADD] medical_medicament_storage: Reusable storage instructions

### PR DESCRIPTION
**Pull request by obulkin**
- Add new model for medicament storage instructions
  (medical.medicament.storage)
- Add views for manipulating medical.medicament.storage records
- Update the storage field on medical.medicament to use the new storage model
  in place of a free-form text field
- Update the form view for medical.medicament to better match the new storage
  field
- Add demo data that showcases the new arrangement

### CODE REVIEW

**Comment by lasley** [comment]
Need to add access rules for all new models - https://travis-ci.org/laslabs/vertical-medical/jobs/167508916#L1604

I'd just mimic the ones in medicament

File: medical_medicament_storage/__openerp__.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+{
+    'name': 'Medical Medicament - Reusable Storage Instructions',
+    'summary': 'Makes storage instructions into reusable entities',
+    'version': '9.0.1.0.0',
+    'category': 'Medical',
+    'website': 'https://laslabs.com/',
+    'author': 'LasLabs, Odoo Community Association (OCA)',
+    'license': 'LGPL-3',
```
Comment (lasley) [inline medical_medicament_storage/__openerp__.py]: AGPL-3

File: medical_medicament_storage/models/medical_medicament.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import fields, models
+
+
+class MedicalMedicament(models.Model):
+    _inherit = 'medical.medicament'
+
+    storage = fields.Many2many(
```
Comment (lasley) [inline medical_medicament_storage/models/medical_medicament.py]: All *2M fields should end in `_ids`

File: medical_medicament_storage/models/medical_medicament_storage.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright 2016 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+from openerp import fields, models
+
+
+class MedicalMedicamentStorage(models.Model):
+    _name = 'medical.medicament.storage'
+    _description = 'Medical Medicament - Storage Instructions'
+
```
Comment (lasley) [inline medical_medicament_storage/models/medical_medicament_storage.py:14]:
```
_sql_constraints = [
    ('name_uniq', 'unique (name)', 'Each name must be unique.'),
]
```

**Review by lasley — CHANGES_REQUESTED** [review]

File: medical_medicament_storage/models/medical_medicament_storage.py (hunk 3 above)
Comment (obulkin) [reply medical_medicament_storage/models/medical_medicament_storage.py:14]: Good call. I didn't realize this was an option.

**Comment by lasley** [comment]
Thanks @obulkin 

FYI the assignment works great in our repos, but in public repos make sure to drop a "Changes finished" or similar comment in order to ping reviewers.

**Review by lasley — APPROVED** [review]

**Pull request merged**